sdk/python/experimental/hello/requirements.txt

```diff
@@ -1 +1 @@
-dagger-io[server]
+dagger-io
```

Comment:
Not sure what this change is :thinking:

Reply:
This is actually needed when installing dagger to work in "extensions", meaning for now it just install Strawberry for generating the GraphQL schema.

Reply:
Yeah I made this change while I was in my "press random buttons to make it work"-mode when trying to get the python entrypoint to running. @helderco are you saying that I should leave this change out? Or that it's correct?

Reply:
I merged as is, let me know if there's any fix needed, will do if so

Reply:
> are you saying that I should leave this change out? Or that it's correct?

Should leave it out. I'll do a follow up.